app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -7,32 +9,52 @@
 import android.view.MenuItem;
 import android.view.View;
 
+import com.google.android.gms.maps.CameraUpdate;
+import com.google.android.gms.maps.CameraUpdateFactory;
 import com.google.android.gms.maps.GoogleMap;
 import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.CameraPosition;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
 
-public class MapsFragment extends SupportMapFragment {
-
-    String map_url;
+import org.fossasia.openevent.dbutils.DbSingleton;
 
+public class MapsFragment extends SupportMapFragment implements LocationListener {
     private GoogleMap mMap;
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setHasOptionsMenu(true);
-
     }
 
     @Override
     public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
+        DbSingleton dbSingleton = DbSingleton.getInstance();
+
+        LatLng latLng = new LatLng(
```

Comment:
why are you getting the details from the db singleton? I think you can pass the necessary db info as a parameter to this view?